### DISCUSSION TITLE
`:G`: can we highlight the files that are currently opened in any buffer?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [tpope/vim-fugitive#2127] [Ideas]
Currently we have:

> Head: master
> Merge: origin/master
> Help: g?
> 
> Unstaged (8)
> M e2e/shipments.go
> M pkg/product/api/mocks/client.go
> M service/inventory/reactor/service_carton_aggregate_events.go
> 

But If I have opened **pkg/product/api/mocks/client.go** - would be nice to see this:

> 
> Head: master
> Merge: origin/master
> Help: g?
> 
> Unstaged (8)
> M e2e/shipments.go
> **M pkg/product/api/mocks/client.go**
> M service/inventory/reactor/service_carton_aggregate_events.go

**Comment by tpope** [comment]
This will obviously vary from user to user, but personally, this doesn't seem very helpful. I normally run with `:set nohidden`, so it's just going to be the files I can see on screen, which _might_ save me a second or 2 here or there when going to look at the patch. If I do have `:set hidden` enabled, it's generally going to be all the files, unless I've recently restarted Vim, in which case it's going to be the files I've opened since restarting. That last one isn't just unhelpful, it sounds actively annoying.

The real deal-breaker, though, is that it would greatly complicate the implementation, as we'd now need to re-render the status window every time a buffer is opened or closed. Even with a fully formed PR, I don't think I would commit to the maintenance burden without a stronger case for the utility of this feature.

Finally, I would note that bold in particular isn't really a practical choice here, as many terminals conflate "bold" with "bright colors", and might not even use a bold font at all.
